resources/known-amendments.md

```diff
@@ -17,12 +17,12 @@ The following is a comprehensive list of all known [amendments](../docs/concepts
 
 | Name                              | Introduced | Status                        |
 |:----------------------------------|:-----------|:------------------------------|
-| [fixAMMOverflowOffer][]           | v2.1.1     | [Open for Voting: 2024-03-27](https://xrpl.org/blog/2024/rippled-2.1.1.html "BADGE_80d0e0") |
+| [fixAMMOverflowOffer][]           | v2.1.1     | [Enabled: 2024-04-11](https://xrpl.org/blog/2024/rippled-2.1.1.html "BADGE_GREEN") |
 | [fixInnerObjTemplate][]           | v2.1.0     | [Open for Voting: 2024-02-20](https://xrpl.org/blog/2024/rippled-2.1.0.html "BADGE_80d0e0") |
 | [fixNFTokenReserve][]             | v2.1.0     | [Open for Voting: 2024-02-20](https://xrpl.org/blog/2024/rippled-2.1.0.html "BADGE_80d0e0") |
 | [DID][]                           | v2.0.0     | [Open for Voting: 2024-01-09](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_80d0e0") |
-| [fixDisallowIncomingV1][]         | v2.0.0     | [Open for Voting: 2024-01-09](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_80d0e0") |
-| [fixFillOrKill][]                 | v2.0.0     | [Open for Voting: 2024-01-09](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_80d0e0") |
+| [fixDisallowIncomingV1][]         | v2.0.0     | [Enabled: 2024-04-11](https://xrpl.org/blog/2024/rippled-2.0.0.html "BADGE_GREEN") |
```

Comment:
+1 to Rome's comment above - please update the URL for fixDisallowIncomingV1 and fixFillOrKill amendments once they are live.